### ISSUE TITLE
Text does not reflow properly and there is 2 dimensional scroll

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#18]
URL: https://hcdev.openplus.ca/en/user/203
User agent: Firefox
Language: both

Issue Description:  Content should be presented without requiring scrolling in two dimensions. Firefox requires horizontal scroll (along with vertical). There is just one word that is cut on this page. 

Steps to reproduce: Set the viewports size to 1024px by 768px 
For Firefox - Open the page in Firefox; In Firefox, go to View in the menu, select Zoom -> Zoom text only. Also test without Zoom text only.
Zoom at 200% and make sure everything is scaled correctly and user has only one dimensional scroll. 

Expected results: There should be only one dimensional scroll. 
![image](https://user-images.githubusercontent.com/14280062/127389587-79d07c6b-372a-4451-b195-02db7151e076.png)

**Comment by qanitah** [comment]
Fixed
